customers/schema/mutations.py

```diff
@@ -164,6 +172,57 @@ def mutate_and_get_payload(cls, root, info, **input):
         return DeleteBoatMutation()
 
 
+class OrganizationInput(graphene.InputObjectType):
+    organization_type = OrganizationTypeEnum(required=True)
+    business_id = graphene.String()
+    name = graphene.String()
+    address = graphene.String()
+    postal_code = graphene.String()
+    city = graphene.String()
+
+
+class BerthServicesInput:
+    invoicing_type = InvoicingTypeEnum()
+    comment = graphene.String()
+
+
+class CreateBerthServicesProfileMutation(graphene.ClientIDMutation):
+    class Input(BerthServicesInput):
+        id = graphene.GlobalID(
+            required=True,
+            description="`GlobalID` associated to the Open City Profile customer profile",
+        )
+        organization = OrganizationInput()
+
+    profile = graphene.Field(ProfileNode)
+    organization = graphene.Field(OrganizationNode)
```

Comment:
not sure this is needed, as `organization` is a field of `PRofileNode` anyway

Reply:
🤔 true... I was following the example of [payments/schema/mutations.py#L415-L420](https://github.com/City-of-Helsinki/berth-reservations/blob/develop/payments/schema/mutations.py#L415-L420)
```python
class UpdateOrderLineMutation(graphene.ClientIDMutation):
    class Input(OrderLineInput):
        id = graphene.ID(required=True)

    order_line = graphene.Field(OrderLineNode)
    order = graphene.Field(OrderNode)
```
But now that I think more about it, it makes more sense to not do it, since profile contains organization and not the other way around.

Reply:
yeah, order can have multiple order lines, so it would require extra effort for the client to find the one that changed, but here only one organization for a profile, so it is rather straightforward.

Reply:
Yup yup, makes sense. 👍